tests/Integration/Behaviour/Features/Context/SpecificPriceRuleFeatureContext.php

```diff
@@ -72,7 +72,7 @@ public function insertSpecificPriceRule($priceRuleName, $type, $value, $minimalQ
     }
 
     /**
-     * @Given /^specific price rule with name (.+) change product price to ([\d\.]+)$/
+     * @Given /^specific price rule with name (.+) change product price to (\d+\.\d+)$/
```

Comment:
Please use `(\d+(\.\d+)?)` instead.

Reply:
Not really, you never say "I spent 10.0€" or "I've finished 50.0% of my job" :/

Reply:
when you do price calculation tests, yes you do lol
I volontary changed this to force float because no one read the manual. Examples given in code must be as usefull as it can

Reply:
But we are not doing price calculation here, we are defining specs :smiley: 
Allowing both (integer and float) will be easier to everyone, and no mistake can happened